docs/src/weights.md

```diff
@@ -50,6 +50,60 @@ w = Weights([1., 2., 3.])
 w = weights([1., 2., 3.])
 ```
 
+### `eweights`
+
+Exponential weights are a common form of temporal weights which assign exponentially decreasing
+weight to past observations.
+
+For each element `i` in `t` the weight value is computed as:
```

Comment:
What is `t`?

Reply:
`t` is just a vector of arbitrary time indices `i` used in the equation below. The reason for this is that it doesn't need to assume ordering or continuity.

Reply:
What I meant is that this should be explained to the reader.